lnwallet/channel.go

```diff
@@ -4393,6 +4393,7 @@ func genHtlcSigValidationJobs(localCommitmentView *commitment,
 			if err != nil {
 				return nil, err
 			}
+
```

Comment:
Ah this commit ended up going away since I needed to move some changes into earlier commits to ensure things compiled. I've removed the commit in the latest diff.